src/main/java/org/opensearch/jobscheduler/rest/action/VisibleForTesting.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler.rest.action;
+
+public @interface VisibleForTesting {
```

Comment:
Doesn't this already exist?
 - `import com.google.common.annotations.VisibleForTesting;` if you have that dependency
 - `import org.opensearch.jobscheduler.utils.VisibleForTesting;` otherwise

Reply:
it does have but it only works when when it is in the same package